test/unit/helpers/ConfigHelper.ts

```diff
@@ -103,10 +106,8 @@ test(`promptOptions 4 - autoRegister = true backwards compatibility for custom i
 
   t.is(finalPromptOptions!.native!.enabled, true);
   t.is(finalPromptOptions!.native!.autoPrompt, true);
-  
-  t.is(finalPromptOptions!.slidedown!.enabled, true);
-  t.is(finalPromptOptions!.slidedown!.autoPrompt, true);
 
+  t.is(finalPromptOptions?.slidedown?.prompts[0].autoPrompt, false);
```

Comment:
This should stay `true`, if the userConfig is just `slidedown: { enabled: true }` then this test is ensuring the default is to autoPrompt.